tests/build_/test_arrangements.py

```diff
@@ -1703,3 +1729,319 @@ def test_export_operator_manifests(self, osbs, extract_platform):
+
+            with open(os.path.join(str(tmpdir), ORCHESTRATOR_OUTER_TEMPLATE), 'r+') as orch_json:
+                build_json = json.load(orch_json)
+                build_json['spec']['triggers'] = imagechange
+
+                orch_json.seek(0)
+                json.dump(build_json, orch_json)
+                orch_json.truncate()
+
+            flexmock(osbs.os_conf, get_build_json_store=lambda: str(tmpdir))
+            (flexmock(BuildRequest)
+                .should_receive('adjust_for_repo_info')
+                .and_return(True))
+
+        additional_params = {
+            'base_image': 'fedora:latest',
+        }
+        if scratch:
+            additional_params['scratch'] = True
+        _, build_json = self.get_orchestrator_build_request(osbs, additional_params)
+        plugins = get_plugins_from_build_json(build_json)
+
+        if scratch:
+            with pytest.raises(NoSuchPluginException):
+                get_plugin(plugins, 'prebuild_plugins', 'check_and_set_rebuild')
+            return
+
+        args = plugin_value_get(plugins, 'prebuild_plugins', 'check_and_set_rebuild', 'args')
+
+        match_args = {
+            "label_key": "is_autorebuild",
+            "label_value": "true",
+        }
+        assert match_args == args
+
+    @pytest.mark.parametrize(('params', 'build_type', 'has_plat_tag',  # noqa:F811
+                              'has_primary_tag'), (
+        ({}, 'orchestrator', False, True),
+        ({'scratch': True}, 'orchestrator', False, False),
+        ({'platform': 'x86_64'}, 'worker', True, False),
+        ({'platform': 'x86_64', 'scratch': True}, 'worker', True, False),
+    ))
+    def test_tag_from_config(self, osbs, params, build_type, has_plat_tag, has_primary_tag):
+        additional_params = {
+            'base_image': 'fedora:latest',
+        }
+        additional_params.update(params)
+        _, build_json = self.get_build_request(build_type, osbs, additional_params)
+        plugins = get_plugins_from_build_json(build_json)
+
+        args = plugin_value_get(plugins, 'postbuild_plugins', 'tag_from_config', 'args')
+
+        assert set(args.keys()) == set(['tag_suffixes'])
+        assert set(args['tag_suffixes'].keys()) == set(['unique', 'primary', 'floating'])
+
+        unique_tags = args['tag_suffixes']['unique']
+        assert len(unique_tags) == 1
+        unique_tag_suffix = ''
+        if has_plat_tag:
+            unique_tag_suffix = '-' + additional_params.get('platform')
+        assert unique_tags[0].endswith(unique_tag_suffix)
+
+        primary_tags = args['tag_suffixes']['primary']
+        if has_primary_tag:
+            assert set(primary_tags) == set(['{version}-{release}'])
+
+    @pytest.mark.parametrize('osbs', [  # noqa:F811
+        {
+            'additional_config': get_pulp_additional_config(),
+            'kwargs': {'registry_uri': 'registry.example.com/v2'},
+        }
+    ], indirect=True)
+    def test_pulp_push(self, osbs):
```

Comment:
The original test has a `# TODO: remove this test after the pulp push plugin is removed`. Is that not relevant anymore?

Reply:
pulp_push has not been removed.

Reply:
Do you know if there are still plans to remove it? Maybe the TODO could be kept there, although I expect the test will start failing anyway if the plugin is removed so it's probably unnecessary.

Reply:
Yes, pulp will be removed soon now that we can use quay.io instead.

Reply:
Ok, thanks, I found the issue that we have filed for it, so the TODO is probably not needed.